InternetHospitalWebUI/src/app/Components/feedbacks/feedbacks.component.html

```diff
@@ -0,0 +1,36 @@
+<div id="GlobalDiv" fxLayout="row" fxLayoutAlign="center center">
+
+  <mat-card id="FatherCard">
+
+    <mat-card-header>
+      <mat-card-title>Leave feedback</mat-card-title>
+    </mat-card-header>
+    <mat-divider></mat-divider>
+    
+    <form [formGroup]="_feedbackService.form" div fxLayout="column" fxLayoutAlign="center center" (ngSubmit)="onSubmit()">
+      
+      <mat-form-field id="SelectComp">
+            <mat-select placeholder="Select type" required formControlName="TypeId" [(value)]="selected">
```

Comment:
Can`t understand, 'selected' isn`t exist in component.ts file, it`s something from angular material?